src/main/java/frc/robot/subsystems/AlgaeIntake.java

```diff
@@ -19,33 +20,36 @@
 
 @Logged
 public class AlgaeIntake extends SubsystemBase {
-  TalonFX intakeMotorOne;
-  TalonFX intakeMotorTwo;
+  TalonFX intakeRollerMotor;
+  TalonFX intakePivotMotor;
   TalonFXConfiguration intakeConfig;
   AngularVelocity intakeHasGamePieceVelocity = constAlgaeIntake.ALGAE_INTAKE_HAS_GP_VELOCITY;;
   Current intakeHasGamePieceCurrent = constAlgaeIntake.ALGAE_INTAKE_HAS_GP_CURRENT;
 
   /** Creates a new AlgaeIntake. */
   public AlgaeIntake() {
-    intakeMotorOne = new TalonFX(mapAlgaeIntake.INTAKE_LEFT_MOTOR_CAN);
-    intakeMotorTwo = new TalonFX(mapAlgaeIntake.INTAKE_RIGHT_MOTOR_CAN);
+    intakeRollerMotor = new TalonFX(mapAlgaeIntake.INTAKE_ROLLER_MOTOR_CAN);
+    intakePivotMotor = new TalonFX(mapAlgaeIntake.INTAKE_PIVOT_MOTOR_CAN);
 
-    intakeMotorOne.getConfigurator().apply(constAlgaeIntake.ALGAE_INTAKE_CONFIG);
-    intakeMotorTwo.getConfigurator().apply(constAlgaeIntake.ALGAE_INTAKE_CONFIG);
+    intakeRollerMotor.getConfigurator().apply(constAlgaeIntake.ALGAE_INTAKE_CONFIG);
+    intakePivotMotor.getConfigurator().apply(constAlgaeIntake.ALGAE_PIVOT_CONFIG);
   }
 
   public boolean hasGamePiece = false;
 
   public void setAlgaeIntakeMotor(double speed) {
-    intakeMotorOne.set(speed);
-    intakeMotorTwo.set(speed);
+    intakeRollerMotor.set(speed);
+  }
+
+  public void setAlgaeIntakePivotPosition(Angle degrees) {
+    intakePivotMotor.setPosition(degrees);
```

Comment:
Degrees as an input implies a unit
```suggestion
  public void setAlgaeIntakePivotPosition(Angle setpoint) {
    intakePivotMotor.setPosition(setpoint);
```